config.toml

```diff
@@ -0,0 +1,11 @@
+baseURL = "https://friendsofdt.org/"
```

Comment:
Not sure how this changes with multiple deployment targets

Reply:
It can be overridden on the command line, so we can make staging deploys point to the right place.